### DISCUSSION TITLE
Help on container configuration

### DISCUSSION THREAD

**Discussion by Alt0car** [qdm12/gluetun#824] [Q&A]
Shit i've deleted my post ...
my question is really simple : i'm using this with cyberghost with gluetun
It is possible to change on demand ip adresse when gluetun is already started ?

**Comment by qdm12** [comment]
I guess this is resolved?

**Reply by Alt0car** [reply]
not the question was already open ^^ i'm trying to change ip without changing my docker-compose conf.
did you have already do that ?

**Reply by qdm12** [reply]
It's rather cumbersome now but you can specify no filter (or large filters such as multiple server hostnames in `SERVER_HOSTNAME`, or multiple regions etc.), and then use the http control server to send a request to stop the vpn and then another request to start it (see the wiki). It'll then pick a vpn connection at random from the pool of connections filtered.

This will be one day much easier to manage via a better API and a mobild UI, but we're still a few months away from this 😉

**Reply by Geniy00** [reply]
@qdm12 Thanks for you answer
it works really good. Moreover, the VPN will be restored automatically in a few seconds, if you didn't call request to start it.

it will be possible to create simple cron task to renew VPN IP every 30 minutes for instance

**Reply by qdm12** [reply]
Yeah on that automatic auto-heal restart, that's kind of bug I need to fix 😄 
Stopping the VPN via the API should keep it stopped 😆
